From 4399a5b6c5adf080657073d6de715d988cc24c74 Mon Sep 17 00:00:00 2001
From: zhangmeng <zhangmeng.kevin@spacemit.com>
Date: Wed, 31 Jan 2024 11:11:43 +0800
Subject: [PATCH 0405/1189] performance optimize

1. change HZ from 100 to 250, to improve realtime response
2. disable KMEMLEAK debug which may make performance worse

Change-Id: Iad956a3805c75fecf20da1f4e77c27788acd3a33
---
 arch/riscv/configs/k1-x_deb1_defconfig | 3 +--
 arch/riscv/configs/k1-x_deb2_defconfig | 3 +--
 arch/riscv/configs/k1-x_evb_defconfig  | 3 +--
 arch/riscv/configs/k1_defconfig        | 3 +--
 4 files changed, 4 insertions(+), 8 deletions(-)

diff --git a/arch/riscv/configs/k1-x_deb1_defconfig b/arch/riscv/configs/k1-x_deb1_defconfig
index ea07e698dca5..48d623a7745e 100644
--- a/arch/riscv/configs/k1-x_deb1_defconfig
+++ b/arch/riscv/configs/k1-x_deb1_defconfig
@@ -29,7 +29,6 @@ CONFIG_SOC_SPACEMIT_K1X=y
 CONFIG_SMP=y
 CONFIG_NR_CPUS=8
 CONFIG_HOTPLUG_CPU=y
-CONFIG_HZ_100=y
 CONFIG_RISCV_SBI_V01=y
 # CONFIG_RISCV_BOOT_SPINWAIT is not set
 CONFIG_PM=y
@@ -522,7 +521,7 @@ CONFIG_CONSOLE_LOGLEVEL_QUIET=8
 CONFIG_DYNAMIC_DEBUG=y
 CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAULT=y
 CONFIG_MAGIC_SYSRQ=y
-CONFIG_DEBUG_KMEMLEAK=y
+CONFIG_DEBUG_FS=y
 CONFIG_DETECT_HUNG_TASK=y
 CONFIG_DEFAULT_HUNG_TASK_TIMEOUT=60
 CONFIG_DEBUG_ATOMIC_SLEEP=y
diff --git a/arch/riscv/configs/k1-x_deb2_defconfig b/arch/riscv/configs/k1-x_deb2_defconfig
index a8bb80232aae..0b56be782b98 100644
--- a/arch/riscv/configs/k1-x_deb2_defconfig
+++ b/arch/riscv/configs/k1-x_deb2_defconfig
@@ -29,7 +29,6 @@ CONFIG_SOC_SPACEMIT_K1X=y
 CONFIG_SMP=y
 CONFIG_NR_CPUS=8
 CONFIG_HOTPLUG_CPU=y
-CONFIG_HZ_100=y
 CONFIG_RISCV_SBI_V01=y
 # CONFIG_RISCV_BOOT_SPINWAIT is not set
 CONFIG_PM=y
@@ -518,7 +517,7 @@ CONFIG_CONSOLE_LOGLEVEL_QUIET=8
 CONFIG_DYNAMIC_DEBUG=y
 CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAULT=y
 CONFIG_MAGIC_SYSRQ=y
-CONFIG_DEBUG_KMEMLEAK=y
+CONFIG_DEBUG_FS=y
 CONFIG_DETECT_HUNG_TASK=y
 CONFIG_DEFAULT_HUNG_TASK_TIMEOUT=60
 CONFIG_DEBUG_ATOMIC_SLEEP=y
diff --git a/arch/riscv/configs/k1-x_evb_defconfig b/arch/riscv/configs/k1-x_evb_defconfig
index d6cf3df8fa65..08702a156d52 100644
--- a/arch/riscv/configs/k1-x_evb_defconfig
+++ b/arch/riscv/configs/k1-x_evb_defconfig
@@ -29,7 +29,6 @@ CONFIG_SOC_SPACEMIT_K1X=y
 CONFIG_SMP=y
 CONFIG_NR_CPUS=8
 CONFIG_HOTPLUG_CPU=y
-CONFIG_HZ_100=y
 CONFIG_RISCV_SBI_V01=y
 # CONFIG_RISCV_BOOT_SPINWAIT is not set
 CONFIG_PM=y
@@ -498,7 +497,7 @@ CONFIG_CONSOLE_LOGLEVEL_QUIET=8
 CONFIG_DYNAMIC_DEBUG=y
 CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAULT=y
 CONFIG_MAGIC_SYSRQ=y
-CONFIG_DEBUG_KMEMLEAK=y
+CONFIG_DEBUG_FS=y
 CONFIG_DETECT_HUNG_TASK=y
 CONFIG_DEFAULT_HUNG_TASK_TIMEOUT=60
 CONFIG_DEBUG_ATOMIC_SLEEP=y
diff --git a/arch/riscv/configs/k1_defconfig b/arch/riscv/configs/k1_defconfig
index 20bea54d79fb..7e72cda06519 100644
--- a/arch/riscv/configs/k1_defconfig
+++ b/arch/riscv/configs/k1_defconfig
@@ -44,7 +44,6 @@ CONFIG_SOC_SPACEMIT_K1X=y
 CONFIG_SMP=y
 CONFIG_NR_CPUS=8
 CONFIG_HOTPLUG_CPU=y
-CONFIG_HZ_100=y
 CONFIG_RISCV_SBI_V01=y
 # CONFIG_RISCV_BOOT_SPINWAIT is not set
 CONFIG_PM=y
@@ -668,7 +667,7 @@ CONFIG_CONSOLE_LOGLEVEL_QUIET=8
 CONFIG_DYNAMIC_DEBUG=y
 CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAULT=y
 CONFIG_MAGIC_SYSRQ=y
-CONFIG_DEBUG_KMEMLEAK=y
+CONFIG_DEBUG_FS=y
 CONFIG_DETECT_HUNG_TASK=y
 CONFIG_DEFAULT_HUNG_TASK_TIMEOUT=60
 CONFIG_DEBUG_ATOMIC_SLEEP=y
-- 
2.47.1

